src/pages/tBTC/Bridge/Minting/MakeDeposit.tsx

```diff
@@ -127,54 +122,38 @@ const MakeDepositComponent: FC<{
         previousStep={MintingStep.ProvideData}
         onPreviousStepClick={onPreviousStepClick}
       />
-      <BridgeProcessCardSubTitle
-        stepText="Step 2"
-        subTitle="Make your BTC deposit"
-      />
-      <BodyMd color="gray.500" mb={6}>
-        Use this generated address to send minimum 0.01&nbsp;BTC, to mint as
-        tBTC.
-      </BodyMd>
-      <BodyMd color="gray.500" mb={6}>
-        This address is a uniquely generated address based on the data you
-        provided.
-      </BodyMd>
       <BTCAddressSection btcDepositAddress={btcDepositAddress} />
-      <Stack spacing={4} mt="5" mb={8}>
-        <BodyMd>Provided Addresses Recap</BodyMd>
-        <AddressRow text="ETH Address" address={ethAddress} />
+      <MintDurationTiers
+        mt={6}
+        items={[
+          {
+            amount: 0.1,
+            rangeOperator: "less",
+            currency: "BTC",
+          },
+          {
+            amount: 1,
+            rangeOperator: "less",
+            currency: "BTC",
+          },
+          {
+            amount: 1,
+            rangeOperator: "greaterOrEqual",
+            currency: "BTC",
+          },
+        ]}
+      />
```

Comment:
No responsiveness in some resoilutions:

Width: 1023
<img width="677" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/d19633c4-31d4-4aab-a0ea-48fe497b9503">

Width: 499
<img width="508" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/e67a3d26-7be2-4016-b5a5-b2abd320202b">